packages/components/src/mobile/readable-content-view/index.native.js

```diff
@@ -3,25 +3,84 @@
  */
 import { View, Dimensions } from 'react-native';
 
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
-const ReadableContentView = ( { reversed, children, style } ) => (
-	<View style={ styles.container }>
-		<View
-			style={ [
-				reversed
-					? styles.reversedCenteredContent
-					: styles.centeredContent,
-				style,
-			] }
-		>
-			{ children }
+const PIXEL_RATIO = 2;
+
+const BREAKPOINTS = {
+	wide: 1024,
+	big: 820,
+	medium: 768,
+	small: 680,
+};
+
+const ReadableContentView = ( { align, reversed, children, style } ) => {
+	const [ windowWidth, setWindowWidth ] = useState(
+		Dimensions.get( 'window' ).width
+	);
+	const [ windowRatio, setWindowRatio ] = useState(
+		Dimensions.get( 'window' ).width / Dimensions.get( 'window' ).height
+	);
```

Comment:
I would suggest destructuring object to get` width` and `height` from `Dimensions`, to simplify a code:

```
const { width, height } = Dimensions.get('window')
```

Reply:
Nice suggestion, it looks cleaner 👍